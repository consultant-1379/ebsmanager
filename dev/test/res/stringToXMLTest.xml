<?xml version="1.0" encoding="UTF-8"?>
<pm xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' xsi:noNamespaceSchemaLocation='PM-MOM.xsd'>
    <pmMimVersion>1.0</pmMimVersion>
    <applicationVersion>6</applicationVersion>
    <measurements>
        <measObjClass name="cell">
            <group name="CounterGroup1">
                <counter>
                    <measType>
                        <measName>c3</measName>
                    </measType>
                    <description/>
                    <storage>
                        <size>numeric(18,0)</size>
                        <aggregation>SUM</aggregation>
                        <type>PEG</type>
                    </storage>
                </counter>
            </group>
        </measObjClass>
    </measurements>
</pm>