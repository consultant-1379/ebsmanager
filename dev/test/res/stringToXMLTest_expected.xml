<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<techpack>
		<Universeclass CLASSNAME="&apos;cell.CounterGroup1&apos;" DESCRIPTION="&apos;&apos;" DiffStatus="" ELEM_BH_REL="0" INHERITANCE="0" OBJ_BH_REL="0" ORDERNRO="0" PARENT="&apos;Computed Counters&apos;" UNIVERSEEXTENSION="&apos;ALL&apos;" VERSIONID="&apos;null&apos;" />
		<Measurementcounter COLNUMBER="100" COUNTAGGREGATION="&apos;SUM&apos;" COUNTERPROCESS="&apos;PEG&apos;" COUNTERTYPE="&apos;PEG&apos;" DATAID="&apos;c3&apos;" DATANAME="&apos;c3&apos;" DATASCALE="0" DATASIZE="18" DATATYPE="&apos;numeric&apos;" DESCRIPTION="&apos;&apos;" DiffStatus="" GROUPAGGREGATION="&apos;SUM&apos;" INCLUDESQL="1" TIMEAGGREGATION="&apos;SUM&apos;" TYPEID="&apos;null&apos;" UNIVCLASS="&apos;&apos;" UNIVOBJECT="&apos;c3&apos;" />
</techpack>